Library/Homebrew/test/test_build_options.rb

```diff
@@ -20,6 +20,7 @@ def test_with_without
     assert @build.with?("bar")
     assert @build.with?("baz")
     assert @build.without?("qux")
+    assert @build.without?("dogs")
```

Comment:
On a serious note, should this be `@build.without?("dog")` rather than `dogs`?

The test below is:
```
assert_equal %w[dog], dep.option_names
```

Very possible I'm misreading the intent there though.